web/src/Pages/TopicPage/TopicCard/TopicCard.test.js

```diff
@@ -20,14 +20,15 @@ describe("Tests for TopicCard using RTL", () => {
     expect(button.textContent).toMatch(/Tell me more/i);
     expect(image.src).toBe(topic.image);
   });
-  test("Renders dummy image if dummy data has no image", () => {
+  test("Renders dummy image if dummy data has no image", async() => {
     // topic.image = false;
     render(
       <AudienceContext>
-        <TopicCard topic={{ ...topic, image: false }} />
+        <TopicCard topic={{ ...topic, image: "noadsdaz" }} />
```

Comment:
I don't think an image prop attribute needs to be passed because the error event would be fired even if a valid `src` for the image is passed due to line 31.
```suggestion
        <TopicCard topic={{ topic }} />
```

Reply:
Is there a need to wrap this in an object? 

```
<TopicCard topic={ topic } />
```

Should work as well?

Reply:
@ChrisOwen101 @s-dilip This is a way of me keeping topic in the same format while also altering the image to check for errors, and also not permanently altering topic for another test later on.